infra/distributor/index.ts

```diff
@@ -379,8 +407,8 @@ export = () => {
     alarmActions,
   });
 
-  new aws.cloudwatch.MetricAlarm(`${serviceName}-stake-alarm-${Severity.SEV2}`, {
-    name: `${serviceName}-log-stake-${Severity.SEV2}`,
+  new aws.cloudwatch.MetricAlarm(`${serviceName}-stake-alarm-${Severity.SEV1}`, {
+    name: `${serviceName}-log-stake-${Severity.SEV1}`,
```

Comment:
Distributor and then a prover running out of stake seems unlikely that it's worth being a sev1, I thought we were avoiding those for our provers?